Modules/@babylonjs/react-native/windows/BabylonReactNative/EngineViewManager.cpp

```diff
@@ -0,0 +1,143 @@
+#include "pch.h"
+#include "EngineViewManager.h"
+
+#include "JSValueReader.h"
+#include "JSValueXaml.h"
+
+using namespace winrt::Microsoft::ReactNative;
+using namespace winrt::Windows::Foundation;
+using namespace winrt::Windows::Foundation::Collections;
+
+using namespace winrt::Windows::UI::Xaml;
+using namespace winrt::Windows::UI::Xaml::Media;
+using namespace winrt::Windows::UI::Xaml::Controls;
+using namespace winrt::Windows::UI::Xaml::Input;
+
+namespace winrt::BabylonReactNative::implementation {
+    EngineViewManager::EngineViewManager() {}
+
+    // IViewManager
+    hstring EngineViewManager::Name() noexcept {
+        return L"EngineView";
+    }
+
+    FrameworkElement EngineViewManager::CreateView() noexcept {
+        _swapChainPanel = SwapChainPanel();
+        _swapChainPanelPtr = nullptr;
+        copy_to_abi(_swapChainPanel, _swapChainPanelPtr);
+        _swapChainPanel.SizeChanged({ this, &EngineViewManager::OnSizeChanged });

```

Comment:
@tymlipari do you have suggestions for how we should be subscribing/auto unsubscribing to these events?